casper/src/main/scala/coop/rchain/casper/api/BlockAPIImpl.scala

```diff
@@ -0,0 +1,626 @@
+package coop.rchain.casper.api
+
+import cats.effect.concurrent.Ref
+import cats.effect.{Concurrent, Sync}
+import cats.syntax.all._
+import com.google.protobuf.ByteString
+import coop.rchain.api.BlockAPI_v2
+import coop.rchain.blockstorage.blockStore.BlockStore
+import coop.rchain.blockstorage.dag.BlockDagStorage
+import coop.rchain.blockstorage.dag.BlockDagStorage.DeployId
+import coop.rchain.blockstorage.deploy.DeployStorage
+import coop.rchain.casper.DeployError._
+import coop.rchain.casper._
+import coop.rchain.casper.api.BlockAPI.{
+  BlockRetrievalError,
+  NoBlockMessageError,
+  ValidatorReadOnlyError
+}
+import coop.rchain.casper.blocks.proposer.ProposeResult._
+import coop.rchain.casper.blocks.proposer._
+import coop.rchain.casper.genesis.contracts.StandardDeploys
+import coop.rchain.casper.protocol._
+import coop.rchain.casper.state.instances.ProposerState
+import coop.rchain.casper.syntax._
+import coop.rchain.casper.util._
+import coop.rchain.casper.util.rholang.{InterpreterUtil, RuntimeManager, Tools}
+import coop.rchain.crypto.PublicKey
+import coop.rchain.crypto.signatures.Signed
+import coop.rchain.graphz._
+import coop.rchain.metrics.{Metrics, Span}
+import coop.rchain.models.BlockHash.BlockHash
+import coop.rchain.models.Validator.Validator
+import coop.rchain.models.rholang.sorter.Sortable._
+import coop.rchain.models.serialization.implicits.mkProtobufInstance
+import coop.rchain.models.syntax._
+import coop.rchain.models.{BlockMetadata, NormalizerEnv, Par}
+import coop.rchain.rspace.hashing.StableHashProvider
+import coop.rchain.rspace.trace._
+import coop.rchain.shared.Log
+import coop.rchain.shared.syntax._
+
+import scala.collection.immutable
+
+class BlockAPIImpl[F[_]: Concurrent: Log: Span: BlockStore: DeployStorage: MultiParentCasper](
+    runtimeManager: RuntimeManager[F],
+    dagStorage: BlockDagStorage[F],
+    validatorOpt: Option[ValidatorIdentity]
+) extends BlockAPI_v2[F] {
+
+  val BlockAPIMetricsSource: Metrics.Source = Metrics.Source(Metrics.BaseSource, "block-api")
+  val DeploySource: Metrics.Source          = Metrics.Source(BlockAPIMetricsSource, "deploy")
```

Comment:
Please start with lowercase for value names (inside class).
https://docs.scala-lang.org/style/naming-conventions.html#constants-values-and-variables

Reply:
[Done](https://github.com/rchain/rchain/pull/3662/commits/e7aeb4416e72d2b82bcc839b87259411d7b66d3b).